Update README.md (#296)
eth-protocol-fellows/protocol-studies: README.md

# Ethereum Protocol Fellowship Study Group (EPFsg)

> This repository is dedicated to fostering coordination and developing a knowledge base for the EPFsg. It hosts the Protocol Wiki, a community effort to build a technical knowledge base focused on the Ethereum core protocol.

Reviewer: ```suggestion
This repository is dedicated to fostering coordination and developing a knowledge base for the EPFsg. It hosts the Protocol Wiki, a community effort to build a technical knowledge base about the Ethereum core protocol.

```
I think "focused on"  implies the wiki has special interest in Ethereum protocol (possibly among other things); about is more direct that Ethereum is entire subject of the wiki.

Learn more about the EPFsg in the [announcement blog post](https://blog.ethereum.org/2024/02/07/epf-study-group).

## Protocol Studies Wiki

You can visit the wiki at https://epf.wiki or head to `/docs` for the source. 

> The wiki is maintained by students of the EPFsg. [Join us](https://discord.com/invite/addwpQbhpq) to learn about the protocol and start [contributing](/docs/contributing.md). 

Reviewer: ```suggestion
The wiki is maintained by students of the EPFsg. [Join us](https://discord.com/invite/addwpQbhpq) to learn about the protocol and start [contributing](/docs/contributing.md). 

```
The wiki is not exclusively maintained by students and is op en to public contribution

Author: I ended up removing that sentence entirely because we state that 

> the Protocol Wiki, [is] a community effort to build a technical knowledge base focused on the Ethereum core protocol.

Because we already explain that the wiki is a community effort it's implied that contributions are open to the public so I don't think it's necessary to explicitly re-state that. Plus we go on to explain this even further in `docs/contributing.md`
